site/styles/_buttons.scss

```diff
@@ -142,10 +140,7 @@
     }
   }
 }
-.s72-btn-play .s72-icon-play path {
-  fill: #fff;
-  stroke: #fff;
-}
```

Comment:
Are we setting these anywhere else? I know we don't want them hardcoded to white for the light theme, but SVGs used to be quite notorious with not cascading styles correctly, so have we tested they work across all the browsers?

Reply:
Tested and working correctly on Chrome, Firefox, IE11, and Safari.